package.json

```diff
@@ -26,18 +28,20 @@
     "react-motion": "^0.5.2",
     "react-redux": "^7.2.1",
     "react-router-dom": "5.2.0",
-    "react-scripts": "3.4.3",
+    "react-scripts": "4.0.3",
     "redux": "^4.1.0",
     "redux-thunk": "^2.3.0",
-    "redux-undo": "1.0.1",
+    "redux-undo": "1.1.0",
+    "sass": "1.45.0",
```

Comment:
Get rid of `node-sass`. `sass` is a JS only implementation of sass. There are newer versions, but 1.45.0 is from when CRA v4 was new.